app/assets/javascripts/components/user_profiles/by_students.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+const ByStudents = React.createClass({
+
+  propTypes: {
+    username: React.PropTypes.string,
+    stats: React.PropTypes.object
+  },
+  render() {
+    return (
+      <div>
+        <h5>
+          {I18n.t('user_profiles.instructors_student_impact', { username: this.props.username })}
+        </h5>
+        <div className= "stat-display">
+          <div className= "stat-display__stat">
+            <div className="stat-display__value">
+              {this.props.stats.word_count}
```

Comment:
'stats.word_count' is not specific enough. It took me a while to figure out from the code whether this is 'by students' word count or 'as student' word count. Maybe in the jbuilder and here, it could be called 'word_count_by_students' or something else that clearly identifies which word count it is.